app/info.md

```diff
@@ -0,0 +1,56 @@
+### Model Description
+
+## Summary
+
+The principle of the estimation is:
+
+1. Augment the number of admissions using the reporting. This is currently done by
+
+ n<sub>aug</sub> = n<sub>reported</sub> / % reported
+ 
+2. Use a log-linear model, parametrised through the doubling time, to simulate epidemic trajectories; see for instance Jombart et al 2020; this is implemented by the RECON package projections.
+
+3. For each admission, simulate duration of hospitalisation from provided distribution.
```

Comment:
Linked to comment below - this is then total hospitalisation not just critical care bed use? Need to check title

Reply:
It can be any type of hospitalisation